source/includes/api/_geostore.md

```diff
@@ -1,58 +1,71 @@
 # Geostore
 
-## What is Geostore?
+***Note — This is the documentation for version 1 of the Geostore microservice, the latest version is [version 2](https://resource-watch.github.io/doc-api/index-rw.html#geoStore-v2).***
 
-Geostore is a geojson database. You can save your geojson or obtain geojson by country, region, Protected areas, etc.
+## What is Geostore
 
-Geostore object contains the following fields:
+Geostore is a database for storing geoStore objects which contain geographic data structures described using the [GeoJSON](https://geojson.org/) schema. GeoJSON supports the following geometry types: `Point`, `LineString`, `Polygon`, `MultiPoint`, `MultiLineString`, and `MultiPolygon`. Geometric objects with additional properties are `Feature` objects. Sets of features are contained by `FeatureCollection` objects.
```

Comment:
I'd suggest something like: 


`A geostore is an object which contain geographic data structures described using the GeoJSON schema. If you are new to GeoJSON, you can read a quick description on its [Wikipedia page](https://en.wikipedia.org/wiki/GeoJSON), or you can read the full specification on its [official page](https://geojson.org/). `

The official page only has the spec, nobody will read that :P

Reply:
Moved this information to concept/_geostore.md. In my opinion, from a GIS user perspective, understanding something about the specifics of GeoJSON and how this relates to the design considerations of Geostore is important. I gave 3 options of varying complexity; overview article focused on developers, links to JSON schema, and the full spec. Also highlighted the important things to consider. see concept/_geostore.md